File: medical-portal/src/API/RSBC.DMF.MedicalPortal.API/ViewModels/Endorsement.cs
namespace RSBC.DMF.MedicalPortal.API.ViewModels
{
    public class Endorsement
    {
        // Pidp User Id (hpdid)
        public string UserId { get; set; }

        // cms loginId
        public Guid LoginId { get; set; }

        // Pidp data
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Role { get; set; }
        public IEnumerable<Licence> Licence { get; set; }
    }

    public class Licence
    {
        public string IdentifierType { get; set; }
        public string StatusCode { get; set; }
        public string StatusReasonCode { get; set; }
    }
}

File: medical-portal/src/UI/src/app/shared/api/models/endorsement.ts
/* tslint:disable */
/* eslint-disable */
import { Licence } from '../models/licence';
export interface Endorsement {
  email?: string | null;
  firstName?: string | null;
  lastName?: string | null;
  licence?: Array<Licence> | null;
  loginId?: string;
  role?: string | null;
  userId?: string | null;
}
